### PR TITLE
ATS-724 Parallel Travis builds

### PR DESCRIPTION
**Pull request by PunchedChimera**
Separating the Transform engine tests into their own Travis Jobs

### CODE REVIEW

**Comment by PunchedChimera** [comment]
Successful snapshot deploy: https://travis-ci.com/github/Alfresco/alfresco-transform-core/builds/160549582
Success release dryrun: https://travis-ci.com/github/Alfresco/alfresco-transform-core/builds/160549600

File: _ci/test.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+pushd "$(dirname "${BASH_SOURCE[0]}")/../"
+
+# Always build the image, but only publish from the "master" branch
+[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && PROFILE="internal" || PROFILE="local"
+
+# If the branch is "master" and the commit is not a Pull Request then deploy the JAR SNAPSHOT artifacts
+[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && DEPLOY="deploy" || DEPLOY="test"
+
+# If the branch is "master" and the profile provided is not "aio-test" do not deploy alfresco-transformer-base
+[ "${1}" != "aio-test" ] && [ "${TRAVIS_BRANCH}" = "master" ] && SKIP_DUP_DEPLOY="-Dparent.deploy.skip=true -Dtransformer.base.deploy.skip=true" || SKIP_DUP_DEPLOY=""
```
Comment (montgolfiere) [inline _ci/test.sh:15]: Why was this needed ?

Comment (PunchedChimera) [reply _ci/test.sh:15]: This was needed in order to prevent deploying multiple alfresco-transform-core and alfresco-transformer-base snapshots. I've discussed this with @eknizat and we feel we have come up with a better solution, separating the deployment of core and base to a separate job.
